cmd/snap-update-ns/secure_bindmount.go

```diff
@@ -21,33 +21,28 @@ package main
 
 import (
 	"fmt"
-	"os"
 	"syscall"
 )
 
 // BindMount performs a bind mount between two absolute paths containing no
 // symlinks, using a private stash directory as an intermediate step.
```

Comment:
This comment probably needs updating now - there is no "private stash directory" anymore, right?